bool state
Robot_Velocity[] robots